From a00210db5dafc938095410baa1f5381fea8fa6fd Mon Sep 17 00:00:00 2001
From: Robert Carr <racarr@google.com>
Date: Wed, 18 Oct 2017 18:46:27 -0700
Subject: [PATCH 008/306] Fix invisible SurfaceViews

Of course OPAQUE != SURFACE_OPAQUE: that would be too easy.

Test: Manual
Bug: 67896876
Change-Id: Ic7aaae42c7de8195c474c39d850f2a07a58ccc9d
(cherry picked from commit 2b8a5e11dc452c11a18eaaab70b6c268d46b8ddd)
---
 core/java/android/view/SurfaceControl.java | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/java/android/view/SurfaceControl.java b/core/java/android/view/SurfaceControl.java
index 54825895733..ff027a947d0 100644
--- a/core/java/android/view/SurfaceControl.java
+++ b/core/java/android/view/SurfaceControl.java
@@ -1161,9 +1161,9 @@ public class SurfaceControl {
          */
         public Transaction setOpaque(SurfaceControl sc, boolean isOpaque) {
             if (isOpaque) {
-                nativeSetFlags(mNativeObject, sc.mNativeObject, OPAQUE, OPAQUE);
+                nativeSetFlags(mNativeObject, sc.mNativeObject, SURFACE_OPAQUE, SURFACE_OPAQUE);
             } else {
-                nativeSetFlags(mNativeObject, sc.mNativeObject, 0, OPAQUE);
+                nativeSetFlags(mNativeObject, sc.mNativeObject, 0, SURFACE_OPAQUE);
             }
             return this;
         }
-- 
2.17.1

